Eatery Blue/AppDelegate.swift

```diff
@@ -34,6 +47,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
 
+    func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
+        // Set APNs token for Firebase Messaging
+        print("APNs device token received: \(deviceToken.map { String(format: "%02.2hhx", $0) }.joined())")
```

Comment:
Same here, replace prints with Logs. Plz do this will all other print statements you use. i want to try to shift away from using extraneous prints